sql/errors.go

```diff
@@ -444,6 +444,12 @@ var (
 	// ErrTableFunctionNotFound is thrown when a table function is not found
 	ErrTableFunctionNotFound = errors.NewKind("table function: '%s' not found")
 
+	// ErrNonAggregatedColumnWithoutGroupBy is thrown when an aggregate function is used with the implicit, all-rows
+	// grouping and another projected expression contains a non-aggregated column.
+	// MySQL error code: 1140, SQL state: 42000
+	ErrNonAggregatedColumnWithoutGroupBy = errors.NewKind("in aggregated query without GROUP BY, expression #%d of SELECT list contains nonaggregated column '%s'; " +
```

Comment:
Do we have an error code mapping for this?

Reply:
Good call; added in https://github.com/dolthub/go-mysql-server/pull/1400.